docs/design/metrics.md

```diff
@@ -69,6 +69,12 @@ How the metrics are exposed to the end-user is determined by the CLI.
 - `transaction_pool.invalid_transactions`: Number of invalid transactions 
 - `transaction_pool.removed_transactions`: Number of removed transactions from the pool
 
+#### Network Manager
+- `network_manager.num_connected_peers`: Number of peers currently connected to
+- `network_manager.pending_session_failures`: Cumulative number of failures of pending sessions
+- `network_manager.total_closed_sessions`: Total number of sessions closed
+- `network_manager.num_incoming_connections`: Total number of incoming connections handled
```

Comment:
the default separator is `_`, so the above metrics will be prefixed with `network_manager_`

Reply:
we should drop the `_manager` part entirely, and just have `network_`